pkg/types/chains/evm/evm.go

```diff
@@ -108,3 +114,51 @@ type Head struct {
 type TransactionFee struct {
 	TransactionFee *big.Int // Cost of transaction in wei
 }
+
+type SignedReport struct {
```

Comment:
Where is this used? We already have a proto, why do we need a Go struct as well?